core/src/i18n/en.yaml

```
@@ -2084,7 +2084,7 @@ top-nav:
   profile:
     preferences: $t(preferences.title)
 upload-yaml:
-  info: You can upload multiple resources, separated in YAML by '---'. If resource already exists, it will be updated, otherwise a new one will be created. In case the Namespace is not given, the 'default' one will be used.
+  info: You can upload multiple resources, separated in YAML by '---'. If resource already exists, it will be updated, otherwise a new one will be created. In case the Namespace is not given, the '{{namespace}}' one will be used.
```

```suggestion
  info: You can upload multiple resources, separated in YAML by '---'. If a resource already exists, it will be updated, otherwise a new one will be created. If the Namespace is not given, the '{{namespace}}' one will be used.
```

`In case` means `as a precaution for if something were to happen`. 
For example, `Take an umbrella in case it rains.` -> this does not mean for you to come back for the umbrella when it starts raining, but to take it as a precaution, JUST in case. (It helps to add this `JUST` to `in case` to validate the right use.) 

What you mean would be reflected by `in the case`, but `if` is much shorter and simpler :) 

See [this post](https://blog.harwardcommunications.com/2013/12/04/how-to-use-in-case/#:~:text=%E2%80%9CIn%20case%20of%E2%80%9D%20(without%20%E2%80%9Cthe%E2%80%9D)%20SHOULD,a%20period%20of%20ten%20years.) for reference.